.travis.yml

```diff
@@ -21,6 +21,11 @@ install:
 
 # activate the environment
 - source activate test-env
+
+# build notifications
+notifications:
```

Comment:
I'm trying to figure out why it's useful to have this in the readme? How does this help developers navigate the application? Could you add some additional detail here?